src/components/toolbox/passphraseInput/index.js

```diff
@@ -127,46 +125,74 @@ class passphraseInput extends React.Component {
+    const {
+      focus,
+      inputsLength,
+      partialPassphraseError,
+      passphraseIsInvalid,
+      showPassphrase,
+      validationError,
+      values,
+    } = this.state;
     const iconName = showPassphrase ? 'showPassphraseIcon' : 'hidePassphraseIcon';
+    const isFeedbackOnError = validationError || secondPPFeedback !== '';
 
     return (
       <React.Fragment>
-        <div className={`${styles.inputs} ${grid.row} passphrase`}>
-          {[...Array(inputsLength)].map((x, i) => (
-            <span key={i} className={`${grid['col-xs-2']}`}>
-              <Input
-                setRef={ref => ref !== null && this.state.focus === i && ref.focus()}
-                placeholder={this.state.focus === i ? '' : i + 1}
-                className={`${this.state.partialPassphraseError[i] || this.state.passphraseIsInvalid || secondPPFeedback !== '' ? 'error' : ''} ${this.state.focus === i ? 'selected' : ''}`}
-                value={values[i] || ''}
-                type={this.state.showPassphrase ? 'text' : 'password'}
-                autoComplete="off"
-                onBlur={this.removeFocusedField}
-                onFocus={this.setFocusedField}
-                onPaste={this.handlePaste}
-                onChange={this.handleValueChange}
-                onKeyDown={this.keyAction}
-                data-index={i}
-              />
-            </span>
-          ))}
-        </div>
-
-        <div className={styles.footerContent}>
+        <div className={styles.wrapper}>
           <label className={`${styles.showPassphrase}`} onClick={this.handleToggleShowPassphrase}>
             <Icon name={iconName} />
-            <span className={`${styles.label}`}>{showPassphrase ? t('Hide passphrase') : t('Show passphrase')}</span>
+            <span className={`${styles.label}`}>{showPassphrase ? t('Hide') : t('Show')}</span>
           </label>
 
-          <Feedback
-            className={styles.errorMessage}
-            show={!!(this.state.validationError || secondPPFeedback !== '')}
-            status={(this.state.validationError || secondPPFeedback !== '') ? 'error' : ''}
-            showIcon
+          <div className={[
+            styles.inputs,
+            partialPassphraseError.length ? styles.boxOnError : '',
+            'passphrase',
+          ].join(' ')}
           >
-            { secondPPFeedback || this.state.validationError }
-          </Feedback>
+            {
+              [...Array(inputsLength)].map((x, i) => (
+                <span key={i}>
+                  <span className={[
+                    styles.inputNumber,
+                    partialPassphraseError[i] ? styles.inputNumberError : '',
+                  ].join(' ')}
+                  >
+                    {`${i + 1}. `}
+                  </span>
+                  <Input
+                    setRef={ref => ref !== null && this.state.focus === i && ref.focus()}
+                    placeholder="___________"
```

Comment:
```suggestion
                    placeholder="___________"
```
🎨  https://www.compart.com/en/unicode/U+005F should make the line connected
![Screenshot 2019-10-18 at 12 02 37](https://user-images.githubusercontent.com/1254342/67085528-428ad680-f19f-11e9-933b-0f9dd20487a7.png)